text/0000-enable-faster-sync.md

```diff
@@ -0,0 +1,81 @@
+
+- Title: enable-faster-sync
+- Authors: [Antioch Peverell](mailto:apeverell@protonmail.com), [John Tromp](mailto:john.tromp@gmail.com)
+- Start date: Oct 25, 2019
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000) 
+- Tracking issue: [Edit if merged with link to tracking github issue]
```

Comment:
Please update this to point to: https://github.com/mimblewimble/grin/issues/3173